bokeh/charts/_builder.py

```diff
@@ -258,64 +221,4 @@ def set_and_get(self, prefix, val, content):
         """
         self._set_and_get(self.data, prefix, self.attr, val, content)
 
-    @property
-    def palette(self):
```

Comment:
Ah, yeap. That sounds good. Just to be sure I got it right, when you say `hasprops` you mean what is discussed in #1819?